config/database.yml

```diff
@@ -0,0 +1,55 @@
+# PostgreSQL. Versions 7.4 and 8.x are supported.
```

Comment:
As for ```database.yml``` - there is convention to keep it in gitignore having just template instead in version control.
Please, rename this file to ``` database.yml.template ```
It is also convinient to add  ```database.yml``` to ```.gitignore``` so everyone will be able to keep their own configuration locally